### PR TITLE
Add Turkish Translation

### PR DESCRIPTION
**Pull request by ardacebi**

### CODE REVIEW

File: resources/locale/tr.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
```
Comment (clarkwinkelmann) [inline resources/locale/tr.yml]: Where does this blank line comes from ?

File: resources/locale/tr.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            disable-user-bio: Kullanıcı açıklama alanını devre dışı bırak
+            options: Seçenekler
+            option-new: Yeni seçenek
+            option-coma-warning: Seçenek açıklaması virgül içeremez (",").
+        types:
+            advanced: Gelişmiş
+            boolean: Checkbox
+            email: E-posta
+            select: Dropdown
+url: Url
```
Comment (clarkwinkelmann) [inline resources/locale/tr.yml]: Indentation is invalid here

**Comment by ardacebi** [comment]
Hi Clark,

I just removed that blank space which I don’t know where it came from :) Anyways, it’s night time here so testing is not possible at the moment, but I will try testing it on my local installation tomorrow.

Thank you for the warnings and for remembering this dead PR lol.

**Comment by clarkwinkelmann** [comment]
It should never have taken so long, but here we go, merged.

I had a quick look around on my test install and everything looks good. Thanks for the contribution :wink:

**Pull request merged**